tuner/examples/punet/README.md

```diff
@@ -0,0 +1,43 @@
+# Punet Tuner
+
+## Environments
+Follow instructions in [`/tuner/README.md`](../README.md)
+
+## Shell Scripts
+
+The required shell scripts can be downloaded from: [sdxl-scripts](https://github.com/nod-ai/sdxl-scripts)
+
+These scripts include:
+1. `compile-punet-base.sh` - Used for compiling model candidates.
+2. `compile_candidate.sh` - Used for compiling dispatch candidates.
+3. `punet.sh` - Invoked by `compile_candidate.sh`.
+
+Please configure the file paths and update commands in `PunetClient`.
+**Note:** Alternatively, add these scripts to your `PATH` environment variable
+
+## Running the Tuner
+
+### [Optional] Generate a tunable mlir
+A sample `2.mlir` is provided for test run. Hoever, this file may become outdated if IREE makes changes to the MLIR format. To ensure you are working with the latest format, please follow the instructions below to compile and generate the most recent benchmark file.
+Use [`punet.sh`](https://github.com/nod-ai/sdxl-scripts/blob/main/tuning/punet.sh) to compile the sample matmul `mmt.mlir` (can also find here: [`mmt_unet.mlir`](https://github.com/nod-ai/sdxl-scripts/blob/main/tuning/mmt_unet.mlir)):
+```
+./punet.sh ./mmt.mlir -o baseline.vmfb --iree-hal-dump-executable-files-to=dump-mmt
+cp ./dump-mmt/module_main_2_dispatch_0_rocm_hsaco_fb_benchmark.mlir ./2.mlir
+```
```

Comment:
Yeah that `2.mlir` is going to get outdated very quickly :P Might be still useful to keep as something to point people at to show them what sort of input the tuner operates on.

I'd be careful overspecializing on external scripts that set dozens of non-default flags, especially if the goal is to turn the tuner into a general purpose tool that supports multiple hardware targets.